context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -30,12 +33,15 @@
     IMAGE_PYRAMID_DIR,
     TILE_REGEX,
     SEQFISH_HYB_CYCLE_REGEX,
-    SEQFISH_FILE_REGEX
+    SEQFISH_FILE_REGEX,
+    CODEX_TILE_DIR
 )
 from .type_client import CommonsTypeClient
 
 
 class SeqFISHViewConf(ImagingViewConf):
+
+    @return_empty_json_if_error
```

Comment:
This is what I was referring to.  This may be too strong, especially given that the entire process in `client.py` is wrapped in a try-catch.

Reply:
I actually prefer this to the try-catch at a higher level.